### ISSUE TITLE
Mealplan, sharing defaults to yourself

### ISSUE THREAD

**Issue by mstrandbo** [TandoorRecipes/recipes#1951]
### Tandoor Version

1.3.2

### Setup

Docker / Docker-Compose

### Reverse Proxy

Nginx Proxy Manager (NPM)

### Bug description

When making an entry in the mealplan, and selecting Sharing, the first name that appears is your own. Why would I need to share with myself? 
Shouldn't my own name be removed from the list? 

Bonus question: Could the mealplan be default shared (with all or selected users) within a space?

**Comment by sebimarkgraf** [comment]
Filtering of the first user sounds sensible.

But I can help with your bonus question. You can set your default share in settings - preferences - plan sharing.

**Comment by mstrandbo** [comment]
> 

Thanks, I hadn't found that setting. That helps a lot :)

**Comment by vabene1111** [comment]
exactly, there should be a filter, there was was one i think but when i moved to generic apis it was removed. Needs to be re-added at some point
